.github/workflows/check-changes-ownership.yml

```
@@ -0,0 +1,60 @@
+name: 'Check changes ownership'
+on:
+  pull_request_target:
+    types: [opened, synchronize]
+    branches: [ 'main*' ]
+    paths:
+      - 'model/**'
+jobs:
+
+  get-changed-files:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    outputs:
+      changed_files: ${{ steps.changed-files.outputs.all_changed_files }}
+      any_changed: ${{ steps.changed-files.outputs.any_changed }}
+    steps:
+       # Checkout the PR branch only to get the changed files from the .git folder
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@24d32ffd492484c1d75e0c0b894501ddb9d30d62 # v47
+        with:
+          dir_names: "true"
+          files: model/**
+          separator: ","
+
+  validate-area-ownership:
+    runs-on: ubuntu-latest
+    needs: get-changed-files
+    permissions:
+      contents: read
+      pull-requests: write
+    if: ${{ needs.get-changed-files.outputs.any_changed == 'true' && github.repository_owner == 'open-telemetry' }}
+    steps:
+      # Checkout the base branch (main, secure context)
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          ref: ${{ github.event.pull_request.base.ref }}
+
+      # uses node 22 to be able to use Typescript
+      - uses: actions/setup-node@v5
+        with:
+          cache: 'npm'
+          cache-dependency-path: |
+            .github/scripts/triage/package-lock.json
+          node-version: '22'
```

I see you've been using `@types/node@24` in `devDependencies` but `22` here. That's not nessarily a problem, but the type-definition may be broader than what's available in `22`.

Aligning them is optimal 🙂 
```suggestion
          node-version: '24'
```